overlord/hookstate/ctlcmd/ctlcmd.go

```diff
@@ -91,20 +91,43 @@ func addCommand(name, shortHelp, longHelp string, generator func() command) {
 	}
 }
 
+type ForbiddenCommandError struct {
+	Message string
+}
+
+func (f ForbiddenCommandError) Error() string {
+	return f.Message
+}
+
+type ForbiddenCommand struct {
```

Comment:
``ForbiddenCommand contains information about an attempt to use a command in a context where it is not allowed.`